integration-cli/docker_api_info_test.go

```diff
@@ -60,6 +65,19 @@ func (s *DockerSuite) TestInfoAPIRuncCommit(c *check.C) {
 	c.Assert(json.Unmarshal(b, &i), checker.IsNil)
 	c.Assert(i.RuncCommit.ID, checker.Not(checker.Equals), "N/A")
 	c.Assert(i.RuncCommit.ID, checker.Equals, i.RuncCommit.Expected)
+
+	// Check the VERSION from the RCE source code match the one of the local binary
```

Comment:
I'd drop this approach and use the vendored VERSION file. See this PR https://github.com/resin-os/balaena/pull/29